plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/SslKafkaAdminFactory.java

```diff
@@ -44,8 +44,8 @@ public SslKafkaAdminFactory(@ForKafkaSsl KafkaAdminFactory delegate, KafkaSslCon
     public Properties configure(ConnectorSession session)
     {
         Properties properties = new Properties();
-        properties.putAll(delegate.configure(session));
         properties.putAll(map);
+        properties.putAll(delegate.configure(session));
```

Comment:
I don't like it, that every implementation of `KafkaAdminFactory ` now has to be updated. Also it affects other entities too (like `KafkaProducerFactory`). Can we load it once in one place so other places would use it?

Reply:
> How about we would make `DefaultTextKafkaConsumerFactory` that would be injected into `PlainTextKafkaConsumerFactory` as delegate (like we do in `SslKafkaConsumerFactory`). The `DefaultTextKafkaConsumerFactory` would not set this property and the `DefaultTextKafkaConsumerFactory` should be default for Kafka. This way it won't override anything, but only load properties from file.
> 
> It would be awkward to explicitly require `PLAINTEXT` in `kafka.properties` that would be overwritten later to something else.

With the proposed solution [here](https://github.com/trinodb/trino/pull/8743#discussion_r681442665l), `SslKafkaXFactory.java` will remain unchanged.